app/controllers/api_controller.rb

```diff
@@ -20,10 +21,18 @@ def current_facility
     @current_facility ||= Facility.find_by(id: request.headers['HTTP_X_FACILITY_ID'])
```

Comment:
We don’t prioritise in V1 because we don’t have the current facility_id information.

Reply:
cool

Reply:
i thought we would be using their last active facility as the current, but makes sense to not implement for v1. I'm assuming v1 doesn't inherit from current.

Reply:
We can't prioritise sync in v1 because we don't have `process_token` and the additional information within to prioritise and batch sycing for current facility and other facilities.

We do have the current facility information even in v1 if the android app sends it in the headers.